glotaran/builtin/file_formats/ascii/wavelength_time_explicit_file.py

```diff
@@ -285,7 +285,7 @@ def get_data_file_format(line):
 def read_ascii_time_trace(fname: str, prepare: bool = True) -> xr.Dataset:
     """Reads an ascii file in wavelength- or time-explicit format.
 
-    See [1] for documentation of this format.
+    See [1]_ for documentation of this format.
```

Comment:
What's the underscore doing here?

Reply:
This is just the proper way to [invoke the citation](https://numpy.org/devdocs/docs/howto_document.html).

https://github.com/glotaran/pyglotaran/blob/3c1c4ae6c8d24e8820144a44d4f3bcc3a0bc5040/glotaran/builtin/file_formats/ascii/wavelength_time_explicit_file.py#L299-L301

One warning less from sphinx 😉

Reply:
Ok, thanks!